host/sgx/windows/entersim.asm

```diff
@@ -1,181 +0,0 @@
-;; Copyright (c) Open Enclave SDK contributors.
-;; Licensed under the MIT License.
-
-include ksamd64.inc
-
-extern __oe_dispatch_ocall:proc
-
-;;==============================================================================
-;;
-;; void oe_enter_sim(
-;;     [IN] void* tcs,
-;;     [IN] uint64_t aep,
-;;     [IN] uint64_t arg1,
-;;     [IN] uint64_t arg2,
-;;     [OUT] uint64_t* arg3,
-;;     [OUT] uint64_t* arg4,
-;;     [OUT] oe_enclave_t* enclave);
-;;
-;; Registers:
-;;     RCX      - tcs: thread control structure (extended)
-;;     RDX      - aep: asynchronous execution procedure
-;;     R8       - arg1
-;;     R9       - arg2
-;;     [RBP+48] - arg3
-;;     [RBP+56] - arg4
-;;
-;; These registers may be destroyed across function calls:
-;;     RAX, RCX, RDX, R8, R9, R10, R11
-;;
-;; These registers must be preserved across function calls:
-;;     RBX, RBP, RDI, RSI, RSP, R12, R13, R14, and R15
```

Comment:
Consider retaining the documentation about registers preserved across function calls.

Reply:
Done.